catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/ProjectListFragment.java

```diff
@@ -292,9 +296,20 @@ public void onClick(DialogInterface dialog, int which) {
 										.commit();
 								break;
 							case 4:
-								Intent intent = new Intent(getActivity(), ProjectUploadActivity.class)
-										.putExtra(ProjectUploadActivity.PROJECT_NAME, item.getName());
-								startActivity(intent);
+								Project chosenProject;
```

Comment:
same as in the other class -> just move the variable into the `try` block as well.